### DISCUSSION TITLE
Exited with code 0xc000007b everytime i initialize a pugi variable

### DISCUSSION THREAD

**Discussion by TheSpectater** [zeux/pugixml#476] [Q&A]
Every time I initialize a pugi variable, it always makes an error. I think my CMake configuration is wrong I'm new with CMake I usually use .sln but not sure.

main.cpp
```c
#include <iostream>
#include <pugixml.hpp>

int main()
{
	pugi::xml_document doc;
	return 0;
}
```

CMakeLists.txt
```
cmake_minimum_required (VERSION 3.20)

project ("Test")

set(CMAKE_CXX_STANDARD 17)

# Add source to this project's executable.
add_executable (CMakeTarget "main.cpp" )

include(FetchContent)
FetchContent_Declare(pugixml
	GIT_REPOSITORY https://github.com/zeux/pugixml.git
	GIT_TAG 08b3433180727ea2f78fe02e860a08471db1e03c) # v1.11.4
FetchContent_MakeAvailable(pugixml)
target_link_libraries(CMakeTarget PRIVATE pugixml::pugixml)
```
Output
`The program '[19996] CMakeTarget.exe' has exited with code 3221225595 (0xc000007b).`

**Comment by zeux** [comment]
I'm not sure off the top of my head why this might happen; to start with you'll need to show the call stack for the exception, and the disassembly at the point where it happens.
